src/Catalyst.Core.Modules.Ledger/Ledger.cs

```diff
@@ -255,5 +281,114 @@ public void Dispose()
             Dispose(true);
             GC.SuppressFinalize(this);
         }
+
+        sealed class ReceiptDeltaTracer : ITxTracer
+        {
+            readonly Delta _delta;
+            readonly long _deltaNumber;
+            readonly Cid _deltaHash;
+            readonly List<TransactionReceipt> _txReceipts;
+            int _currentIndex;
+
+            public ReceiptDeltaTracer(Delta delta, Cid deltaHash)
+            {
+                _delta = delta;
+                _deltaHash = deltaHash;
+                _deltaNumber = delta.DeltaNumber;
+                _txReceipts = new List<TransactionReceipt>(delta.PublicEntries.Count);
+            }
+
+            public IEnumerable<TransactionReceipt> Receipts => _txReceipts;
+
+            public bool IsTracingReceipt => true;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private property 'IsTracingReceipt'.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4979613)